File: 23.sum_of_unknown_number_of_intgers/sum_of_unknown_number_of_intgers.py
sum_of_all_unknown_number_of_integers = 0
print("Please enter as many integers as you want on seperate lines followed by control+d(on mac) or control+z(on windows)")
while (True):
  try:
    input_integer=int(input())
    sum_of_all_unknown_number_of_integers+=input_integer
  except EOFError:
    break
print("The sum of all numbers you entered is :", sum_of_all_unknown_number_of_integers)

File: 23.sum_of_unknown_number_of_intgers/sum_of_unknown_number_of_intgers.cpp
#include <iostream>

int main(){
  int sum_of_all_unknown_number_integers=0,input_integer=0;
  std::cout<<"Please enter as many integers as you want followed by control+d(on mac) or control+c(on windows)"<<std::endl;
  while(std::cin>>input_integer){
    sum_of_all_unknown_number_integers+=input_integer;
  }
  std::cout<<"The sum of all numbers you entered is " <<sum_of_all_unknown_number_integers<<std::endl;
  return 0;
}
